reporter/bin/check.py

```diff
@@ -77,6 +77,9 @@
 reports += ChatLogsSource().query('uncaughtException', threshold=1)
 reports += ChatLogsSource().query('SyntaxError', threshold=1)
 
+# @see https://kibana5.wikia-inc.com/app/kibana#/discover/b4a08460-3633-11e7-8beb-4b869586dfbf?_g=(time%3A(from%3Anow-6h%2Cmode%3Aquick%2Cto%3Anow))
+reports += PHPExecutionTimeoutSource(period=21600).query(threshold=5)
```

Comment:
This is just a guess, but I'm not convinced that these errors will always be closely grouped. Do you think that a longer period will be a problem?